boards/arm/stm32h7/nucleo-h743zi/Kconfig

```diff
@@ -27,4 +27,66 @@ config STM32_ROMFS_IMAGEFILE
         depends on STM32_ROMFS
         default "../../../rom.img"
 
+
+config PROGMEM_OTA_PARTITION
+	bool "Progmem ota partitions"
+	default y
+	select MTD
+	select MTD_BYTE_WRITE
+	select MTD_PARTITION
+	select MTD_PROGMEM
+	select MTD_PROGMEM_ERASESTATE
+
+menuconfig APP_FORMAT_MCUBOOT
+	bool "MCUboot-bootable format"
+	default y
+	select PROGMEM_OTA_PARTITION
+	---help---
+		The MCUboot support of loading the firmware images.
+
+if APP_FORMAT_MCUBOOT
+
+config MCUBOOT_BOOTLOADER
+	bool "MCUboot bootloader application"
+	default y
```

Comment:
Please consider if this can be defaulted to `n`.

Reply:
I have problems with this configurations. It inverts the state of my MTD configurations.